list-component/list.css

```diff
@@ -0,0 +1,29 @@
+@import url(/common.css);
+
+.list{
+    border: 1px solid black;
+    width: 12rem;
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacing7);
+    /* align-items: center; */
```

Comment:
You can remove the comment if not needed.

Reply:
i will do it